APITesters/ObjCAPITester/ObjCAPITester/RCStoreProductAPI.m

```diff
@@ -49,4 +49,24 @@ + (void)checkAPI {
       );
 }
 
++ (void)checkSubscriptionType {
+    RCStoreProductSubscriptionType type = RCStoreProductSubscriptionTypeSubscription;
+
+    switch (type) {
+        case RCStoreProductSubscriptionTypeSubscription: break;
+        case RCStoreProductSubscriptionTypeInAppPurchase: break;
```

Comment:
thoughts on just naming this nonsubscription? i always found the "inApp" terminology a little confusing...they can both be in the app

Reply:
Oooh great point, and technically all of these are IAPs I guess. I'll change it.